server/services/store/sqlstore/board.go

```diff
@@ -384,6 +406,8 @@ func (s *SQLStore) saveMember(db sq.BaseRunner, bm *model.BoardMember) (*model.B
 		"scheme_viewer":    bm.SchemeViewer,
 	}
 
+	oldMember, _ := s.getMemberForBoard(db, bm.BoardID, bm.UserID)
```

Comment:
This is not amazing, but is the simplest approach that I found to detect if this is an insert or an update.

Reply:
fair point

Reply:
Done.